libs/csm/rest/csm_rest_iamuser.py

```diff
@@ -587,7 +587,47 @@ def get_iam_user_rgw(self, uid, header):
                                           headers=header)
         self.log.info("Get IAM user request successfully sent...")
         return response
+    
+    @RestTestLib.authenticate_and_login
+    def get_iam_user_payload_rgw(self, payload: dict, auth_header=None):
```

Comment:
These functions can be clubbed to existing functions with auth_header flag where ever it is not there.

Reply:
@amit-si-seagate  I agree with @kachhwahadivya  we needs to have only 1 interfacting API function to be consumed rom out of CSM components. The new added methods if they don't exists can become private/protected methods (prefix  __ or  _) .

Reply:
This comment applies to all newly added methods.

Reply:
Resolved @kachhwahadivya